src/components/Footer/Footer.js

```diff
@@ -1,7 +1,32 @@
 import React from 'react';
+import styled from 'styled-components';
+
+const FooterStyle = styled.div`
+  .Footer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100px;
+    border: 1px gray solid;
+    color: rgb(134, 133, 133);
+  }
+`;
 
 const Footer = () => {
-  return <div>footer</div>;
+  return (
+    <FooterStyle>
+      <div className="Footer">
+        <p>
+          <strong>주식회사 Sucream </strong>
+          <br />
+          <br />
+          <span>주소: 서울특별시 강남구 테헤란로 427</span>
+          <span> | </span>
+          <span>공동 대표: 강현구, 양대영, 유호진, 이정석, 진민욱</span>
+        </p>
+      </div>
+    </FooterStyle>
```

Comment:
```suggestion
    <FooterStyle>
        <p>
          <strong>주식회사 Sucream </strong>
          <br />
          <br />
          <span>주소: 서울특별시 강남구 테헤란로 427</span>
          <span> | </span>
          <span>공동 대표: 강현구, 양대영, 유호진, 이정석, 진민욱</span>
        </p>
    </FooterStyle>
```
style-components 사용시에는 className 사용은 지양해주세요! 다음 리뷰 참고해주세요!

Reply:
네 감사합니다.

1번 수정 완료했습니다.
2번 수정 완료했습니다.
3번 네 반영하겠습니다.
4번 브랜치 분리 완료했습니다.